Microsoft.Azure.Cosmos/src/Resource/Container/ContainerCore.Items.cs

```diff
@@ -432,18 +432,27 @@ internal abstract partial class ContainerCore : ContainerInternal
         public override IOrderedQueryable<T> GetItemLinqQueryable<T>(
             bool allowSynchronousQueryExecution = false,
             string continuationToken = null,
-            QueryRequestOptions requestOptions = null)
+            QueryRequestOptions requestOptions = null,
+            CosmosPropertyNamingPolicy? cosmosPropertyNamingPolicy = null)
         {
             requestOptions ??= new QueryRequestOptions();
 
+            CosmosSerializationOptions serializationOptions = 
```

Comment:
The current logic will break if new enum value is added. 
```suggestion
            CosmosSerializationOptions serializationOptions = this.ClientContext.ClientOptions.SerializerOptions;
            if (cosmosPropertyNamingPolicy.HasValue)
            {
                 serializationOptions = new CosmosSerializationOptions
                {
                    PropertyNamingPolicy = cosmosPropertyNamingPolicy.Value;
                }
            }
```